src/current/cockroachcloud/connect-to-your-cluster.md

```diff
@@ -21,8 +21,8 @@ By default, CockroachDB {{ site.data.products.dedicated }} clusters are locked d
 
 - Allowed IP address ranges on the internet.
 - Cloud-provider-specific peer networking options:
-    - Google Cloud Platform (GCP) VPC Peering
-    - Amazon Web Services (AWS) Private link
+    - Google Cloud Platform (GCP) VPC Peering or Private Service Connect (Preview)
```

Comment:
later on I'm seeing there are anchors these could link to: `#gcp-private-service-connect` etc